bids2openminds/main.py

```diff
@@ -10,22 +10,23 @@
 
 from .utility import table_filter, pd_table_value, file_hash, file_storage_size
 from .mapping import bids2openminds_instance
-from . import globals
 
 
 def create_techniques(layout_df):
     suffixs = layout_df["suffix"].unique().tolist()
     techniques = []
     not_techniques_index = ["description", "participants", "events"]
     for suffix in suffixs:
-        #excluding the None and non thechnique indexes
+        # excluding the None and non thechnique indexes
         if not (pd.isna(suffix) or (suffix in not_techniques_index)):
-            openminds_techniques_cache=bids2openminds_instance(suffix, "MAP_2_TECHNIQUES")
-            #Excluding the suffixs that are not in the library or flagged as non technique suffixes
+            openminds_techniques_cache = bids2openminds_instance(
+                suffix, "MAP_2_TECHNIQUES")
+            # Excluding the suffixs that are not in the library or flagged as non technique suffixes
             if not pd.isna(openminds_techniques_cache):
                 techniques.extend(openminds_techniques_cache)
             else:
-                warn(f"The {suffix} suffix is currently considered an auxiliary file for already existing techniques or a non technique file.")
+                warn(
```

Comment:
? why is there a line break?

Reply:
The toolbox that I am using to make it PEP8 compliant keep adding this. I don't know why :/ 
I thought, I have deleted it.

Reply:
it makes the line length fit within the limit